docs/products/opensearch/howto/migrate-ism-policies.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Reapply ISM policies after snapshot restore
+sidebar_label: Reapply ISM policies
+limited: true
+---
+
+Reapply Index State Management (ISM) policies to Aiven for OpenSearch® using a script.
+
+After restoring your snapshot, ISM policies that manage index rollover, retention, and
+deletion must be reapplied to your indices. These policies are stored in the
+`.opendistro-ism-config` index, but the assignments between indices and policies must
+be reapplied using a script.
+
+## What is restored
+
+The `.opendistro-ism-config` index stores ISM policy configurations and is restored with
+the snapshot, but policy assignments to specific indices are stored in the cluster
+metadata and must be reapplied.
+
+## Prerequisites
+
+- A machine with network access to Aiven for OpenSearch services
+- Python 3.11 or higher installed
+- Ensure all data indices and the `.opendistro-ism-config` index are restored from the
+  snapshot
+- The snapshot was restored with `include_global_state: true`
+
+:::warning
+
+- **Snapshot must include global state**
+  Ensure the snapshot was restored with `include_global_state: true`. If it was
+  restored without the global state, the ISM policy assignments are not be available in the
+  cluster metadata, and the script fails to reapply the policies.
+
+- **Script can only be run once**
+  OpenSearch clears the cluster metadata with ISM policy assignments after the policies
+  are applied. This means the script can only be run once.
+
+:::
+
+## Reapply ISM policies
+
+The script retrieves the ISM policy assignments stored in the cluster state and
+reapplies them to the corresponding indices.
+
+To reapply ISM policies to indices in Aiven for OpenSearch:
+
+1. Download the script from the
+   [Aiven examples GitHub repository](https://github.com/aiven/aiven-examples/blob/main/solutions/reapply-ism-policies/avn-re-apply-ism-policies.py).
+
+1. Create a JSON configuration file with the connection details for your Aiven for
+   OpenSearch service. The following is an example configuration file:
+
+   ```json
+   {
+       "host": "target-ip-or-fqdn",
+       "port": target-port-number,
+       "user": "the user",
+       "password": "the password"
+   }
+   ```
+
+   Use the default `aivenadmin` user for this task.
```

Comment:
I'm not sure I understand this instruction, is it related to line 58, or to this task in general (line  41) ?

Reply:
Good catch. It is the 'user' in the config file. Updated step to clarify this.